cdap-ui/app/hydrator/templates/partial/node-config-modal/preview-tab.html

```diff
@@ -108,41 +108,48 @@ <h3 class="text-center">
 
     <h4>Output Records</h4>
 
-    <div ng-if="HydratorPlusPlusNodeConfigCtrl.previewData.output.records.length > 0">
-      <!-- OUTPUT RECORDS -->
-      <table class="table table-bordered">
-        <thead>
-          <th ng-repeat="field in ::HydratorPlusPlusNodeConfigCtrl.previewData.output.schema">
-            {{ ::field }}
-          </th>
-        </thead>
-
-        <tbody>
-          <tr ng-repeat="row in ::HydratorPlusPlusNodeConfigCtrl.previewData.output.records">
-            <td ng-repeat="field in ::HydratorPlusPlusNodeConfigCtrl.previewData.output.schema">
-              {{ ::row[field] }}
-            </td>
-          </tr>
-        </tbody>
-      </table>
-    </div>
+    <div ng-repeat="(key, value) in HydratorPlusPlusNodeConfigCtrl.previewData.output">
```

Comment:
I believe this is causing this.

![screen shot 2017-11-14 at 4 59 10 pm](https://user-images.githubusercontent.com/1452845/32813114-5c98695a-c95d-11e7-9056-f72bb070ca61.png)